src/components/PostInfo/PostInfo.spec.jsx

```diff
@@ -62,22 +62,22 @@ describe('PostInfo', () => {
       cy.get('.UserInfo').should('contain', 'Ervin Howell');
     });
 
-    it('should contain all comments', () => {
-      cy.get('.CommentInfo').should('have.length', 3);
-      cy.get('.CommentInfo__name')
-        .eq(0)
-        .should('have.text', 'esse autem dolorum');
-      cy.get('.CommentInfo__name')
-        .eq(2)
-        .should(
-          'have.text',
-          'culpa eius tempora sit consequatur neque iure deserunt',
-        );
-    });
-
-    it('should not show NoCommentsMessage', () => {
-      cy.getByDataCy('NoCommentsMessage').should('not.exist');
-    });
+    // it('should contain all comments', () => {
+    //   cy.get('.CommentInfo').should('have.length', 3);
+    //   cy.get('.CommentInfo__name')
+    //     .eq(0)
+    //     .should('have.text', 'esse autem dolorum');
+    //   cy.get('.CommentInfo__name')
+    //     .eq(2)
+    //     .should(
+    //       'have.text',
+    //       'culpa eius tempora sit consequatur neque iure deserunt',
+    //     );
+    // });
+
+    // it('should not show NoCommentsMessage', () => {
+    //   cy.getByDataCy('NoCommentsMessage').should('not.exist');
+    // });
```

Comment:
These tests are commented out. If they are needed, consider uncommenting them to ensure the `PostInfo` component is thoroughly tested for displaying comments and the absence of the 'NoCommentsMessage'.